### PR TITLE
[FIX] project: Make 'Sub-tasks' feature editable on projects

### PR DESCRIPTION
**Pull request by ktr-odoo**
If user enables 'sub-tasks' in settings,
if project is fsm project('is_fsm' is True) then 'sub-tasks' is set to False
for those projects. otherwise it is set to True for all those project.
User can not disable/enable it.it was not possible to enable or disable sub-tasks
feature from project form view.
task-2192763

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/project/models/project.py [hunk 1]
```diff
@@ -216,7 +216,7 @@ def _get_default_rating_status(self):
     date = fields.Date(string='Expiration Date', index=True, tracking=True)
     subtask_project_id = fields.Many2one('project.project', string='Sub-task Project', ondelete="restrict",
         help="Project in which sub-tasks of the current project will be created. It can be the current project itself.")
-    allow_subtasks = fields.Boolean('Sub-tasks', compute='_compute_allow_subtasks')
+    allow_subtasks = fields.Boolean('Sub-tasks', compute='_compute_allow_subtasks', store=True, readonly=False)
```
Comment (tivisse) [inline addons/project/models/project.py]: This field will never be recomputed, as there is no trigger. So the stored value could be wrong.

Comment (mba-odoo) [reply addons/project/models/project.py]: @tivisse The compute method _compute_allow_subtasks is already overrided in the industry_fsm and over there the dependency is added for the "is_fsm".

Thanks

Comment (tivisse) [reply addons/project/models/project.py]: Yes, but it means that if you only install project, this is wrong ;)

Comment (mba-odoo) [reply addons/project/models/project.py]: @tivisse , Right 👍 , I have done the changes accordingly. Thanks

**Pull request closed without merging**